pyproject.toml

```diff
@@ -50,6 +50,7 @@ django-tailwind = "^3.5.0"
 django-ratelimit = "^4.0.0"
 twitterapi = "^2.8.2"
 django-htmx = "^1.14.0"
+django-ses = "^3.4.1"
```

Comment:
Hm, should this be here?

Reply:
No. Sorry, I  messed up the order of the dependencies when I fixed the merge conflict with the `main` branch. `django-ses` should be in line 52 and `django-htmx` in line 53.